### PR TITLE
Adding create client table

### PR DESCRIPTION
**Pull request by Criofann**
Adding the first create client table iteration to the main path so the rest of the team can work on it.
This addition allows for the creation of the client table as well as some example data that is being entered into the table.

Proposed solution for point 3 of the product backlog "As a member of the Sales team I want to be able to create a new client. I should be able to store a name, address and phone number"

### CODE REVIEW

**Comment by lullabelle** [comment]
Good informative commit comments!

**Review by lullabelle — CHANGES_REQUESTED** [review]
This looks great!
Couple of points just - keep the code indentations consistent
nice use of naming convention in the title!

File: clienttable.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
+USE JCAC_CormacF;
+DELIMITER $$
+DROP PROCEDURE IF EXISTS my_client_table $$
+CREATE PROCEDURE my_client_table()
```
Comment (jkelso662) [inline clienttable.sql]: Again, discuss this with your team. You need to at least be consistent with the rest of the project when naming your procedures!

File: clienttable.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    -- Insert a new record into the "Client" table
+    INSERT INTO Client (Name,Address,Phone)
+        VALUES ('Shaun Gauley','4-6 Upper Crescent, Belfast','028 9057 1100'), ('Denise McShaw','Lisburn Road','028 7748 4523'), ('Flora Sava','Carneybaun Estate','028 9073 2278'),
+        ('Peter Blair','York Street','028 0094 3853'), ('Mark Landing','Ballybogey','028 8746 32983'), ('Fiona Carey','Smithfield Lane','028 8535 3254'), 
+        ('Theresa August','Willowbrook Park','028 7486 4320'), ('Mary Gilbert','Princess Street','028 7473 9347'), ('Matthew Smith','Corbyfield Park','028 0274 3439'),
+        ('William McDonald','Sherbert Lane','028 4376 8203');
+
+
+    -- Check the number of affected rows 
+    GET DIAGNOSTICS @rows = ROW_COUNT;
```
Comment (jkelso662) [inline clienttable.sql]: Tabulation is all over the show here!

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Same as other MR. Agree a naming convention for your procedures and fix the tabs. Correct tabs are one of the easiest things to get right and will increase my happiness and therefore the likelihood I will approve significantly!

**Review by jkelso662 — APPROVED** [review]
Good job :)))))))

**Review by lullabelle — APPROVED** [review]
approved

**Pull request merged**